packages/botonic-react/src/webchat/header.jsx

```diff
@@ -19,7 +19,7 @@ const Header = styled(Flex)`
   z-index: 2;
 `
 
-const ImageContainer = styled(Flex)`
+const ImageContainer = styled.div`
```

Comment:
Is `display: flex;` missing here?

Reply:
I think it's not necessary, but I'll review